core/src/display_object/interactive.rs

```diff
@@ -577,30 +576,8 @@ pub trait TInteractiveObject<'gc>:
             Avm1::run_stack_frame_for_method(self_do, object, context, method_name, &[other]);
         } else if let Avm2Value::Object(object) = self_do.object2() {
             let mut activation = Avm2Activation::from_nothing(context.reborrow());
-            let event_name = if focused {
-                "focusIn".into()
-            } else {
-                // `focusOut` is not this simple in FP,
-                // firing it might break SWFs that rely
-                // on the specific behavior
```

Comment:
@Lord-McSweeney Not sure what this meant, from my testing it seems that `focusOut` simply works that way